### PR TITLE
edits to index and faq

### PR DESCRIPTION
**Pull request by craigmateo**
Simple text changes to docs index and faq pages.

Signed-off-by: Frayne, Craig <craig.frayne@intel.com>

### CODE REVIEW

File: faq.md [hunk 1]
```diff
@@ -68,17 +53,13 @@ There are different ways partners can contribute to this project:
 * Build the infrastructure to support OPEA projects 
 
 ## Where can partners see the latest draft of the Conceptual Framework spec? 
-A version of the spec is available in the docs repo in this project 
+A version of the spec is available in the documentation repository within this project.
```
Comment (dbkinder) [inline faq.md:56]: The name of the repo is "docs", so we should include mentioning this.

**Review by dbkinder — CHANGES_REQUESTED** [review]
Need to add Signed-off-by (via git commit -s --amend)
One comment to address, otherwise LGTM

File: faq.md (hunk 1 above)
Comment (craigmateo) [reply faq.md]: added "docs" and link to repo

**Comment by craigmateo** [comment]
updated using PR #89

**Pull request closed without merging**